build_all.sh

```diff
@@ -20,12 +20,13 @@ conda install -yq boa
 # load gpuci tools
 source ~/.bashrc
 
-conda mambabuild -c ${CONDA_USERNAME:-rapidsai} -c ${NVIDIA_CONDA_USERNAME:-nvidia} -c conda-forge --python=$PYTHON  \
-    recipes/xgboost
+export RAPIDS_CONDA_BLD_ROOT_DIR='/tmp/conda-bld-workspace'
+export RAPIDS_CONDA_BLD_OUTPUT_DIR='/tmp/conda-bld-output'
```

Comment:
```suggestion
export RAPIDS_CONDA_BLD_OUTPUT_DIR='/tmp/conda-bld-output'

env  | sort
```

Adding `env` is sometimes useful for debugging. Might as well add this since we're touching this file.